virtual/jack/jack-1.ebuild

```diff
@@ -13,4 +13,5 @@ RDEPEND="
 	|| (
 		media-sound/jack2[${MULTILIB_USEDEP}]
 		media-sound/jack-audio-connection-kit[${MULTILIB_USEDEP}]
+		media-video/pipewire[${MULTILIB_USEDEP},jack-sdk(-)]
```

Comment:
Why `(-)`?

Reply:
This indicates that if the jack-sdk IUSE is absent from the ebuild (such as from versions before 0.3.26 that first made it available), then the package manager must treat it as if it was disabled when resolving dependencies. (+) would mean it's enabled if absent. Without any of them it may produce unsolvable reverse dependencies and is a CI failure. Thanks to @a17r for explaining this.

Reply:
But this is wrong: If `USE=jack-sdk` for `media-video/pipewire` is not available, that pipewire ebuild does **not** satisfy `virtual/jack`.

Reply:
I fail to see how that would be wrong - virtual/jack only accepts media-video/pipewire[jack-sdk]. If its either disabled [-jack-sdk] or missing, then such configurations will not satisfy the dependency and instead pull in one of the existing JACK servers. In fact, got hit by just that when doing the 0.3.27 bump - no jack-sdk IUSE for that ebuild led to refusal by Portage to install such a package. As expected.

Reply:
While that's true (this isn't `(+)`), I don't see the point of `(-)` here. Just `[...,jack-sdk]` is enough, portage already knows that a missing flag doesn't sastisfy a requested flag. This just makes it look more confusing.

afaik `(-)` only makes sense when disabling with `[-flag(-)]` given a missing flag wouldn't satisfy -flag otherwise.

Reply:
That makes sense. Thanks, Ionen. I guess I misunderstood Andreas and why CI was failing.

Reply:
Without "(+)" or "(-)", we would get MissingUseDepDefault error for new virtual because <media-video/pipewire-0.3.27 has no USE=jack-sdk. Specifying "jack-sdk(-)" like you did is correct: Like written in devmanual (https://devmanual.gentoo.org/general-concepts/dependencies/#use-dependency-defaults), it sets a default in case the flag does not exist in the target package.

Sorry!